padl/transforms.py

```diff
@@ -1031,7 +1033,8 @@ def eval_apply(self, inputs: Iterable, flatten: bool = False, **kwargs):
         return self._pd_itercall(inputs, 'eval', loader_kwargs=kwargs,
```

Comment:
I would add `use_post` as an option to `eval_apply` as well. It's possible users may want to calculate embedding vectors in bulk instead of returning output of the post processing.